complete/src/main/java/com/example/restservice/dao/CustomQuestions.java

```diff
@@ -0,0 +1,63 @@
+package com.example.restservice.dao;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class CustomQuestions {
+
+    @JsonProperty("questions")
+    List<Question> questions;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public static class Question {
+        @JsonProperty("questionId")
```

Comment:
I'm wondering if we're using our own custom representation of the questions object. Then we'll need to write our own parsing library to generate the forms on UI.
Maybe we should try an existing library like
https://github.com/rjsf-team/react-jsonschema-form ?